context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FilePairSystemMonitor.java

```diff
@@ -209,18 +201,20 @@ protected final void startWatcher() {
 
             @Override
             public void run() {
-                this.watchService = FileSystems.getDefault().newWatchService();
+                try {
+                    this.watchService = FileSystems.getDefault().newWatchService();
+                } catch (IOException e) {
+                    e.printStackTrace();
```

Comment:
Use logger.

And shouldn't there be some kind of handling the exception besides logging it?

If `newWatchService` throws then `this.watchService` will be `null`, which will cause an NPE in `watchedPath.register(this.watchService,...`

Or maybe it is ok to remove the whole `try-catch` here because the call of `startWatcher` is already wrapped in a `try-catch` in `de.atb.context.monitoring.monitors.file.FilePairSystemMonitor#run` (line 121).

Comment also applies to the other two file monitor classes.

Reply:
logger is now used